examples/catalog/lib/expansion_tile_sample.dart

```diff
@@ -43,17 +45,20 @@ final List<Entry> data = <Entry>[
       Entry('Section A2'),
     ],
   ),
-  Entry('Chapter B',
+  Entry(
+    'Chapter B',
     <Entry>[
       Entry('Section B0'),
       Entry('Section B1'),
     ],
   ),
-  Entry('Chapter C',
+  Entry(
+    'Chapter C',
     <Entry>[
       Entry('Section C0'),
       Entry('Section C1'),
-      Entry('Section C2',
+      Entry(
+        'Section C2',
         <Entry>[
           Entry('Item C2.0'),
           Entry('Item C2.1'),
```

Comment:
these changes are probably ok, though i can imagine situations where it'd be less ideal